main/pipeline/pipeline.cc

```diff
@@ -1020,7 +1019,7 @@ ast::ParsedFilesOrCancelled typecheck(unique_ptr<core::GlobalState> &gs, vector<
                         (*preemptionManager)->tryRunScheduledPreemptionTask(*gs);
                     }
                 }
-                if (cancelable && epochManager->wasTypecheckingCanceled()) {
+                if (cancelable && epochManager.wasTypecheckingCanceled()) {
```

Comment:
Ok, so this used to be `ctx.state.epochManager` and now it's `gs->epochManager`?

I'm not sure I understand the distinction...are they the same, or different?

Reply:
Discussed via Tuple; `ctx.state` is a reference to `gs` (see where it gets created above in method).